packages/teleport-plugin-react-app-routing/src/index.ts

```diff
@@ -31,8 +37,10 @@ export const createPlugin: ComponentPluginFactory<AppRoutingComponentConfig> = (
 
   const reactAppRoutingComponentPlugin: ComponentPlugin = async (structure) => {
     const { uidl, dependencies, options } = structure
+    // @ts-ignore-next-line
+    const { useFolderStructure, disableDOMInjection, flavour } = options.meta || {}
 
-    registerRouterDeps(dependencies)
+    RouteDependencies[flavour](dependencies)
```

Comment:
you can have an if statement here, later a switch if needed, it is a bit more clear, you don't have to shift attention to another place to check which possible flavors exist